Source/Core/Runtime/Configuration/BaseRuntimeConfiguration.cs

```diff
@@ -91,6 +91,7 @@ public virtual UnityEngine.InputSystem.InputActionAsset CurrentInputActionAsset
         public IModeHandler Modes { get; protected set; }
 
         /// <inheritdoc />
+        [Obsolete("This property is obsolete and no longer returns a valid value. Use LocalUser instead.")]
```

Comment:
```suggestion
        [Obsolete("This property is obsolete and no longer returns a valid value. Use LocalUser instead.", true)]
```
Suggesting to make it an error warnings are easily overlooked and you log an error for DefaultRuntimeConfiguration.User as well,